neuraxle/steps/loop.py

```diff
@@ -146,6 +161,36 @@ def should_resume(self, data_container: DataContainer, context: ExecutionContext
         return False
 
 
+class ContinueException(Exception):
+    pass
+
+
+class BreakException(Exception):
+    pass
+
+
+class Break(Identity):
+    def __init__(self, condition_function: Callable):
+        Identity.__init__(self)
+        self.condition_function = condition_function
+
+    def _did_process(self, data_container: DataContainer, context: ExecutionContext) -> DataContainer:
+        if self.condition_function(data_container):
+            raise BreakException()
+        return data_container
+
+
+class Continue(Identity):
```

Comment:
```suggestion
class ContinueWhen(Identity):
```

Reply:
Or maybe "ContinueIf" or "ContinueOn". I just suggest that it could be more verbose maybe.

Reply:
ContinueIf